doc/user_guide/user_guide.md

```diff
@@ -466,6 +466,138 @@ CREATE OR REPLACE JAVA SET SCRIPT IMPORT_FILES(...) EMITS (...) AS
+parallel running importer processes. These parallel processes can be controlled
+by setting the `PARALLELISM` parameter.
+
+By default, this parameter is set to the `nproc()`. The `nproc()` is an Exasol
+special SQL command that returns the total number of data nodes in your cluster.
+
+```sql
+IMPORT INTO <schema>.<table>
+FROM SCRIPT CLOUD_STORAGE_EXTENSION.IMPORT_PATH WITH
+  BUCKET_PATH     = 's3a://<S3_PATH>/*'
+  DATA_FORMAT     = 'ORC'
+  S3_ENDPOINT     = 's3.<REGION>.amazonaws.com'
+  CONNECTION_NAME = 'S3_CONNECTION'
+  PARALLELISM     = 'nproc()';
+```
+
+In the example above, `PARALLELISM` property value is set to `nproc()` which
+returns the number of physical data nodes in the cluster. Thus, the storage
+extension starts `nproc()` many parallel importer processes. The total number of
+files is distributed among these processes in a round-robin fashion and each
+process imports data from their own set of files.
+
+However, you can increase the parallelism by multiplying it with a number. For
+example, in order to start four times more processes, set it:
+
+```sql
+PARALLELISM = 'nproc()*4'
+```
+
+Or to a higher static number as `PARALLELISM = '16'` that will use 16 importer
+processes in total.
+
+We **recommend** to set the parallelism properly depending on the cluster
+resources (number of cores, memory per node), the number of files and the size
+of each file.
+
+### Export Parallelism Parameter
+
+In the export, the parallelism works differently compared to the import SQL
+statement.
+
+In the import statement, we are importing data from many files. Using the user
+provider parallelism number, we distribute these files into that many importer
+processes. For example, simply by taking modulo of file hash by parallelism
+number.
+
+In export, we have a table with many records. When exporting an Exasol table,
+the `PARALLELISM` parameter value is internally used in a `GROUP BY` clause to
+distribute the table records into many exporter processes. The parallelism
+should be something dynamic that Exasol database can understand and use in the
+group by clause.
+
+The default value for parallelism for export is `iproc()` (notice that it is
+different from `nproc()`). It returns the data node id numbers. Therefore,
+by default, it creates as many exporter processes as the number of datanodes.
+
+```sql
+EXPORT <schema>.<table>
+INTO SCRIPT CLOUD_STORAGE_EXTENSION.EXPORT_PATH WITH
+  BUCKET_PATH     = 's3a://<S3_PATH>/'
+  DATA_FORMAT     = 'PARQUET'
+  S3_ENDPOINT     = 's3.<REGION>.amazonaws.com'
+  CONNECTION_NAME = 'S3_CONNECTION'
+  PARALLELISM     = 'iproc()';
+```
+
+Like in import, you can increase the number of exporter processes. Since we need
+a dynamic number that the Exasol database can understand, you can combine the
+`iproc()` statement with `random()` and `floor()` operations.
+
+For example, to increase the exporter processes four times, set it as below:
+
+```sql
+PARALLELISM = 'iproc(), floor(random()*4)'
```

Comment:
I once discussed with Torsten, that
```sql
`iproc(),mod(rownum,4)'
```
should also work but have somewhat better performance (less calculation).

Reply:
This is also great option, I am changing to this as suggested less calculation indeed.